src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogBuildListener.java

```diff
@@ -107,7 +110,7 @@ public void onStarted(Run run, TaskListener listener) {
 
             logger.fine("End DatadogBuildListener#onStarted");
         } catch (Exception e) {
-            DatadogUtilities.severe(logger, e, "An unexpected error occurred: ");
+            DatadogUtilities.severe(logger, e, null);
```

Comment:
why `null` ?

Reply:
null is being replaced by the save sentence within the severe method now. This is to have more consistency and less copy past everywhere